cms/tests/test_rendering.py

```diff
@@ -52,7 +91,7 @@ class RenderingTestCase(CMSTestCase):
 
     def setUp(self):
         super(RenderingTestCase, self).setUp()
-        self.test_user = self._create_user("test", True, True)
+        self.test_user = self.get_superuser()
```

Comment:
What is the reason that this user was changed?

Reply:
`_create_user("test", True, True)` is the same as `get_superuser()`. get_superuser is easier to understand so I changed that.

Reply:
So it does. Agreed better to use the util.